arm-batch/2017-01-01/swagger/BatchManagement.json

```diff
@@ -1150,10 +1151,15 @@
         "id": {
```

Comment:
@ravbhatnagar It's not inside the property bag, it's a second level property inside a `KeyVaultReference` which is inside a properties bag.

Take a look at the example json which Xing provided [here](https://github.com/xingwu1/azure-rest-api-specs/blob/fa606e173348f3013cfc391fe9a722030743f316/arm-batch/2017-01-01/examples/BatchAccountCreate_BYOS.json#L16).